mapie/conformity_scores/residual_conformity_scores.py

```diff
@@ -24,28 +24,33 @@ def __init__(
 
     def get_signed_conformity_scores(
         self,
+        X: ArrayLike,
         y: ArrayLike,
         y_pred: ArrayLike,
     ) -> NDArray:
         """
-        Compute the signed conformity scores from the predicted values
-        and the observed ones, from the following formula:
+        Compute the signed conformity scores from the observed values
+        and the estimator, from the following formula:
         signed conformity score = y - y_pred
         """
         return np.subtract(y, y_pred)
 
     def get_estimation_distribution(
         self,
+        X: ArrayLike,
         y_pred: ArrayLike,
-        conformity_scores: ArrayLike,
-    ) -> NDArray:
+        values: ArrayLike
```

Comment:
UPD: according to the comments on the `conformity_score.py` file, prefer to use `conformity_scores`.